-1 
Tested at: e2e22b37b8c0574323e8a26a539a37ef7a1a3798
 I found an error when building: 
 
 <pre>>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATPhotonProducer.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATPhotonSlimmer.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATPrimaryVertexCleaner.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATSecondaryVertexSlimmer.cc 
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc: In member function 'virtual void pat::PATPackedCandidateProducer::produce(edm::Event&, const edm::EventSetup&)':
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc:167:21: error: 'const VertexRef' has no member named 'isNonNull'
           if(PVOrig.isNonNull()) PV = reco::VertexRef(PVs, PVOrig.key()); // WARNING: assume the PV slimmer is keeping same order
                     ^
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATSingleVertexSelector.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATTauProducer.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-04-22-2300/src/PhysicsTools/PatAlgos/plugins/PATTauSlimmer.cc 
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8842/4356/summary.html

--- a/PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc
+++ b/PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc
@@ -163,7 +163,7 @@ void pat::PATPackedCandidateProducer::produce(edm::Event& iEvent, const edm::Eve
           math::XYZPoint vtx = cand.vertex();
           pat::PackedCandidate::LostInnerHits lostHits = pat::PackedCandidate::noLostInnerHits;
           const reco::VertexRef & PVOrig = associatedPV[reco::CandidatePtr(cands,ic)];
-          if(PVOrig.isNonNull()) PV = reco::VertexRef(PVs, PVOrig.key()); // WARNING: assume the PV slimmer is keeping same order
+          if(PVOrig.isNonnull()) PV = reco::VertexRef(PVs, PVOrig.key()); // WARNING: assume the PV slimmer is keeping same order
           int quality=associationQuality[reco::CandidatePtr(cands,ic)];
 //          if ((size_t)pvi!=PVOrig.key()) std::cout << "not closest in Z" << pvi << " " << PVOrig.key() << " " << cand.pt() << " " << quality << std::endl;
           //          TrajectoryStateOnSurface tsos = extrapolator.extrapolate(trajectoryStateTransform::initialFreeState(*ctrack,&*magneticField), RecoVertex::convertPos(PV->position()));

